### PR TITLE
feat: all campaigns page new features

### PR DESCRIPTION
**Pull request by luzzif**

### CODE REVIEW

File: packages/frontend/src/components/campaigns/campaign/index.tsx [hunk 1]
```diff
@@ -1,39 +1,43 @@
-import type { AmmInfo } from "@/src/types";
 import { Apr, SkeletonApr } from "./apr";
 import { Pool, SkeletonPool } from "./pool";
 import { SkeletonStatus, Status } from "./status";
 import { Rewards, SkeletonRewards } from "./rewards";
 import { Chain, SkeletonChain } from "./chain";
 import type { NamedCampaign } from "@/src/hooks/useCampaigns";
 
+import styles from "./styles.module.css";
+import Link from "next/link";
```
Comment (guerrap) [inline packages/frontend/src/components/campaigns/campaign/index.tsx:9]: Import here.

File: packages/frontend/src/components/campaigns/campaign/styles.module.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.root {
+    composes: row from "../styles.module.css";
+    @apply bg-white 
+        hover:bg-gray-200
+        cursor-pointer
+        transition-colors
+        duration-200
+        rounded-xl h-14;
```
Comment (guerrap) [inline packages/frontend/src/components/campaigns/campaign/styles.module.css:8]: I'd add the `ease-in-out` class here.

File: packages/frontend/src/ui/pagination/styles.module.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+.arrowLeft {
+    @apply transform rotate-90;
+}
+
+.arrowRight {
+    @apply transform -rotate-90;
+}
+
+.item {
+    @apply w-9 h-9 rounded-md flex items-center justify-center transition-colors duration-200;
```
Comment (guerrap) [inline packages/frontend/src/ui/pagination/styles.module.css:18]: `ease-in-out` too.

File: packages/frontend/src/ui/pagination/styles.module.css [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+.itemActive {
+    @apply bg-gray-200 hover:bg-gray-200;
+}
+
+.clickable {
+    @apply cursor-pointer hover:bg-gray-200 transition-colors duration-200;
```
Comment (guerrap) [inline packages/frontend/src/ui/pagination/styles.module.css:26]: Same as above.

File: packages/frontend/src/components/campaigns/campaign/index.tsx (hunk 1 above)
Comment (luzzif) [reply packages/frontend/src/components/campaigns/campaign/index.tsx]: Addressed in e3a2263d1d5914abd841a5695dcc92f0d2856e17

File: packages/frontend/src/components/campaigns/campaign/styles.module.css (hunk 2 above)
Comment (luzzif) [reply packages/frontend/src/components/campaigns/campaign/styles.module.css:9]: Addressed in e3a2263d1d5914abd841a5695dcc92f0d2856e17

File: packages/frontend/src/ui/pagination/styles.module.css (hunk 3 above)
Comment (luzzif) [reply packages/frontend/src/ui/pagination/styles.module.css]: Addressed in e3a2263d1d5914abd841a5695dcc92f0d2856e17

File: packages/frontend/src/ui/pagination/styles.module.css (hunk 4 above)
Comment (luzzif) [reply packages/frontend/src/ui/pagination/styles.module.css]: Addressed in e3a2263d1d5914abd841a5695dcc92f0d2856e17

**Review by guerrap — APPROVED** [review]

**Pull request merged**